### DISCUSSION TITLE
115网盘无法上传文件

### DISCUSSION THREAD

**Discussion by qinguoluo** [AlistGo/alist#3812] [Q&A]
alist挂载115网盘后，能够正常浏览网盘内的内容，能够播放视频。能够新建文件夹，但是不能新建文件和上传文件，提示升级到最新版本: unexpected error。alist是最新版v3.13.2。不知道这是什么原因，请高手们赐教。

**Comment by charleycy** [comment]
同问，也遇到了

**Comment by windmagic1980** [comment]
同问，上传时出现

请升级到最新版本: unexpected error

的提示

![image](https://user-images.githubusercontent.com/64878285/225605310-bdb2d509-9ccd-423f-94df-ff4b6a6d3dd9.png)

**Reply by windmagic1980** [reply]
使用的版本为 Docker 版 xhofe/alist-aria2 版本号为 v3.13.2

**Comment by qinguoluo** [comment]
115网盘可以上传文件了。alist升级到最新版 3.14.0。
